### DISCUSSION TITLE
435

### DISCUSSION THREAD

**Discussion by giscus** [Shane-Park/markdownBlog#172] [Comments]
# 435

Intro 스프링부트에서는 아래와 같은 설정으로 간단하게 인메모리 H2 데이터베이스를 사용 할 수 있습니다. application.yml spring: datasource: url: jdbc:h2:mem:testdb username: sa build.gradle.kts runtimeOnly("com.h2database:h2") 이후 h2:console 설정을 켠다면, 콘솔 페이지에서 데이터베이스를 직접 확인 해 볼 수 있는데요 spring: h2: console: enabled: true settings: web-allow-others: true 위와 같이 설정했을때에는, /h2-console 경로로 들어가면 데이터베이스를 웹 페이지에서 직접 확인 해 볼 수 있습니다. 그런데, Test Connectio..

https://shanepark.tistory.com/435

**Comment by millwheel** [comment]
덕분에 완전 간편하게 해결했습니다. 감사합니다.
